framework/include/base/MeshGeneratorSystem.h

```diff
@@ -221,4 +237,7 @@ class MeshGeneratorSystem : public PerfGraphInterface, public libMesh::ParallelO
 
   /// Whether any of the mesh generators are a \p BreakMeshByBlockGenerator
   bool _has_bmbb;
+
+  /// Holds the ordering of generators based on mesh dependencies
+  std::vector<std::vector<std::string>> _ordered_generators;
```

Comment:
you already have this in the attributes  std::vector<std::vector<MeshGenerator *>> _ordered_mesh_generators;
no need to duplicated?

Reply:
Ah didn't realize there was another variable for this. Actually looking into when `_ordered_mesh_generators` is set, it only gets defined in the `execute_mesh_generators` action, whereas the logic in ReactorMeshParams to set `bypass_meshgen` needs access to this ordering in `create_added_mesh_generators` action (one action earlier). Both `std::vector<std::vector<std::string>> ordered_generators` and `_ordered_mesh_generators` get defined differently so I'm not sure if there is a way to reuse the same definition for both.

Reply:
good point. 

So let s keep what you have but move it above the other one in the header.
And say in the docstring that it s only valid from this point to this point. Notably, I dont know that it s the same order as the other vector.